DKTDDSwiftUI/Extensions/GreetingsExtension.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  GreetingsExtension.swift
+//  DKTDDSwiftUI
+//
+//  Created by Ramon Geronimo on 10/2/22.
+//
+
+import Foundation
+
+extension Greetings {
+    // MARK: Helpers
+    /// This helper function helps when array of names has two values and properly format the result. Ex: ["Jon", "Peter"] -> "Jon, and Peter"
+    /// It also checks if the array input has a value that has a string with two names separated with comma. Ex: ["Jon, Peter", "Cris"]
+    /// - Parameters:
+    ///   - names: String array containing two names
+    /// - Returns: result, a string with names formatted 
+    func handleNamesCountEqualsTwo(_ names: [String]) -> String {
+        var result = ""
+        if names[0].contains(",") {
+            let newNames = names[0].components(separatedBy: ", ")
+            result += "\(newNames[0]), \(newNames[1]), and \(names[1])"
+        } else if names[1].contains(","){
+            let newNames = names[1].components(separatedBy: ", ")
+            result += "\(names[0]), \(newNames[0]), and \(newNames[1])"
+        } else {
+            result += "\(names[0]), and \(names[1])"
```

Comment:
I think for all of these you can just return instead of appending result then returning it at the end